app/assets/javascripts/discourse/app/models/topic-tracking-state.js

```diff
@@ -83,20 +83,50 @@ const TopicTrackingState = EmberObject.extend({
    *
    * @method establishChannels
    */
-  establishChannels() {
-    this.messageBus.subscribe("/latest", this._processChannelPayload);
+  establishChannels(meta) {
```

Comment:
Small improvement, if you do `meta = {}` here then you don't need to check for meta everywhere below, you can just do:

```
meta["/latest"] || -1
```

Reply:
Or if we are worried about `PreloadStore` passing null here, we could just do this as the first thing in the function:

```
meta ??= {}
```